docs/utils/iers.rst

```diff
@@ -7,23 +7,40 @@ IERS data access (`astropy.utils.iers`)
 Introduction
 ============
 
-The `~astropy.utils.iers` package provides access to the tables provided by
-the International Earth Rotation and Reference Systems (IERS) service, in
-particular files allowing interpolation of published UT1-UTC and polar motion
-values for given times.  The UT1-UTC values are used in `astropy.time` to
-provide UT1 values, and the polar motions are used in `astropy.coordinates` to
-determine Earth orientation for celestial-to-terrestrial coordinate
-transformations.
+The `~astropy.utils.iers` package provides access to the tables provided by the
+`International Earth Rotation and Reference Systems <https://www.iers.org/>`_
+(IERS) service, in particular the `Earth Orientation data
+<https://www.iers.org/IERS/EN/DataProducts/EarthOrientationData/eop.html>`_
+allowing interpolation of published UT1-UTC and polar motion values for given
+times.  The UT1-UTC values are used in :ref:`astropy-time` to provide UT1 values, and
+the polar motions are used in `astropy.coordinates` to determine Earth
+orientation for celestial-to-terrestrial coordinate transformations.
 
 .. note:: The package also provides machinery to track leap seconds.  Since it
           generally should not be necessary to deal with those by hand, this
           is not discussed below.  For details, see the documentation of
           `~astropy.utils.iers.LeapSeconds`.
 
+There are two IERS data products that are commonly used in the :ref:`astropy-time`
```

Comment:
They are used both in `time` and `coordinates`; maybe just remove the reference here?